.travis.yml

```diff
@@ -2,32 +2,52 @@ language: c
 dist: trusty
 sudo: false
 cache: ccache
+
 env:
-- ARCH=x64
+ global:
+  - ARCH=x64
+
+addons:
+  apt:
+    packages:
+    # required by openssl installer
+    - perl
+
 matrix:
  include:
  - name: "Linux GCC: +centos +debian"
    os: linux
    compiler: gcc
-   env: ADDITIONAL_BUILDS="centos debian"
+   env: ADDITIONAL_BUILDS="centos debian" LINKAGE=std
    before_script:
     - ./configure --install-deps --disable-lz4-ext --prefix="$PWD/dest"
  - name: "Linux clang: +alpine"
    os: linux
    compiler: clang
-   env: ADDITIONAL_BUILDS="alpine"
+   env: ADDITIONAL_BUILDS="alpine" LINKAGE=std
    before_script:
     - ./configure --install-deps --disable-lz4-ext --prefix="$PWD/dest"
+ - name: "Linux clang: +static +alpine-static"
+   os: linux
+   compiler: clang
+   env: ADDITIONAL_BUILDS="alpine-static" LINKAGE=static
+   before_script:
+    - ./configure --enable-static --install-deps --source-deps-only --disable-gssapi --disable-lz4-ext --prefix="$PWD/dest"
  - name: "OSX GCC"
    os: osx
    compiler: gcc
+   env: LINKAGE=std
    before_script:
     - ./configure --install-deps --disable-lz4-ext --prefix="$PWD/dest"
  - name: "OSX clang: +static"
    os: osx
+   # Use an older image to and disable syslog for broader compatibility
+   # with old and new osx versions.
+   osx_image: xcode9.2
```

Comment:
Is this still necessary with the `--disable-syslog` flag?

Reply:
Yes, older toolchains complained of .a files being generated by newer toolchains.